tests/endpoints/echo/client/client.go

```diff
@@ -26,78 +26,32 @@ import (
 	"time"
 
 	"github.com/GoogleCloudPlatform/esp-v2/tests/env/testdata"
+	"github.com/GoogleCloudPlatform/esp-v2/tests/utils"
+	tpoauth2 "github.com/GoogleCloudPlatform/esp-v2/third_party/golang_internal"
```

Comment:
Nit: Pre-existing go code in our repository prefers putting all named imports in a separate block, not merged into unnamed imports

Reply:
Still needs to be fixed